README.md

```diff
@@ -1,19 +1,88 @@
-### thanks for 100 stars :)
-
 ```
  _   _____   __   ____  ___  ___   _  ________                
 | | / / _ | / /  / __ \/ _ \/ _ | / |/ /_  __/__________  ____
 | |/ / __ |/ /__/ /_/ / , _/ __ |/    / / / /___/ __/ _ \/ __/
 |___/_/ |_/____/\____/_/|_/_/ |_/_/|_/ /_/     /_/ / .__/\__/ 
                                                   /_/         
 ```
+[![Discord][discord-shield]][discord-url]
+[![License][license-shield]][license-url]
+[![Stars][stars-shield]][stars-url]
+[![Forks][forks-shield]][forks-url]
+[![Releases][releases-shield]][releases-url]
+[![Language][language-shield]][language-url]
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>  
+    <li><a href="explanation">Explanation</a></li>
+    <li><a href="#installation">Installation</a></li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#support">Support</a></li>
+    <li><a href="#disclaimer">Disclaimer</a></li>
+  </ol>

```

Comment:
Can I ask why whole readme is in Table of Contents too? If you will close ToC it will not show readme.
```suggestion
<details open="open">
  <summary>Table of Contents</summary>
  <ol>  
    <li><a href="explanation">Explanation</a></li>
    <li><a href="#installation">Installation</a></li>
    <li><a href="#usage">Usage</a></li>
    <li><a href="#license">License</a></li>
    <li><a href="#support">Support</a></li>
    <li><a href="#disclaimer">Disclaimer</a></li>
  </ol>
</details>
```

Reply:
I forgot to add it